pkg/kubehound/core/core_ingest_local.go

```diff
@@ -4,16 +4,20 @@ import (
 	"context"
 	"fmt"
 	"os"
+	"path/filepath"
 
+	"github.com/DataDog/KubeHound/pkg/collector"
 	"github.com/DataDog/KubeHound/pkg/config"
+	"github.com/DataDog/KubeHound/pkg/dump"
 	"github.com/DataDog/KubeHound/pkg/ingestor/puller"
 )
 
 func CoreLocalIngest(ctx context.Context, khCfg *config.KubehoundConfig, resultPath string) error {
 	// Using the collector config to ingest the data
 	khCfg.Collector.Type = config.CollectorTypeFile
 	// Applying the clusterName from the dynamic config (from CLI or env var) to the collector config
-	khCfg.Collector.File.ClusterName = khCfg.Dynamic.ClusterName
+	// khCfg.Collector.File.ClusterName = khCfg.Dynamic.ClusterName
```

Comment:
is this wanted? (should it remove the comment altogether instead?)

Reply:
yes will remove the comment also